.gitignore

```diff
@@ -73,3 +73,10 @@ target/
 
 # IDE specific configurations
 .idea/
+
+# npm
+node_modules/
+package-lock.json
+
+# commitlint
+commitlint.config.js
```

Comment:
Why is this ignored?

Reply:
You're right. I added this to the .gitignore before realizing we will need this file in the pipeline as well.

Reply:
Fixed